nobrainerzoo/cli.py

```diff
@@ -178,12 +195,25 @@ def predict(
         model_cmd = eval(spec["command"])
     except NameError:
         model_cmd = spec["command"]
+        
+    if container_type == "singularity":
+        bind_paths = ",".join(bind_paths)
+        cmd_options = ["--nv", "-B", bind_paths, "-B", f"{out_path}:/output", "-W", "/output"]
+        cmd = ["singularity", "run"] + cmd_options + [image] + model_cmd.split() \
+            + model_options
+    elif container_type == "docker":
+        bind_paths_docker = []
+        for el in bind_paths:
+            bind_paths_docker += ["-v", f"{el}:{el}"]
+        cmd_options = bind_paths_docker + ["-v", f"{out_path}:/output", "-w", "/output", "--rm"]
+        cmd = ["docker", "run"] + cmd_options + [image] + model_cmd.split() \
+            + model_options
 
```

Comment:
could you raise an error if the container_type is not docker or singularity?

```suggestion
        else:
            raise ValueError(f"unknown container type: {container_type}")
```